.github/workflows/go.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: 1.20
```

Comment:
can we do smth like https://github.com/nspcc-dev/locode-db/commit/6975c2e0e90ec5310cf39dd18ed634e369bd3b49? it will allow us to not change this line in the future

Reply:
Great advise, thanks!